lightwood/helpers/templating.py

```diff
@@ -68,6 +67,10 @@ def is_allowed(v):
 
 
 def call(entity: dict) -> str:
+    # Special behavior for ensemble
+    if 'submodels' in entity['args']:
```

Comment:
Why is this needed?

From what I can gather, it removes all submodels from the JsonAI representation once built (well, `model.args.mixers = $mixers`). Is it intended that we remove mixers from the IR once predictor code is generated? I'd argue not.